### ISSUE TITLE
Edit layout buttons is not accessible using keyboard present on edit popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#141]
User Experience:

User rely on keyboard/Screen Reader will find difficulty to interact with the controls if controls are not accessible through keyboard as user won't be able to perform functionality over such control.

 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the Pages window. Select "APIs: List" option.

6. Navigate to the any editable control details and select edit layout.

7. Navigate to the various controls of the "Edit Layout" dialog box.

8. Verify all the controls are accessible or not using keyboard.

 

Actual Result:

Edit layout buttons is not accessible using keyboard, present on edit popup. This type of popup appearing for few duration.  

 
Expected Result:

Edit layout buttons should be accessible using keyboard, present on edit popup.

 

MAS Reference:

MAS 2.1.1 - Keyboard (36)

Note:

Similar issue exist with Chrome.

Similar issue exist throughout the application, for this type of popup controls as ‘save , discard’ etc..

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
 
![image](https://user-images.githubusercontent.com/2320302/79778189-35d17180-82ed-11ea-84ee-e0fa53572759.png)

**Comment by azaslonov** [comment]
Fixed.
